src/components/Seat/JetsSeat.test.js

```diff
@@ -0,0 +1,448 @@
+import { render, screen } from '@testing-library/react';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import {
+  paramsData,
+  seatDataFirst,
+  seatDataBusiness,
+  seatDataPremium,
+  seatDataEconomy,
+  seatDataAisle,
+  seatDataEmpty,
+  seatDataIndex,
+} from './__fixtures__';
+import { JetsSeat } from './index';
+
+const setup = ({ data = {}, config = {}, params = {} } = {}) => ({
+  ...render(
+    <MockJetsContextProvider config={config} params={{ ...paramsData(params) }}>
+      <JetsSeat data={data} />
+    </MockJetsContextProvider>
+  ),
+});
+
+describe('JetsSeat', () => {
+  describe('when any seat type is rendered', () => {
+    it('should add the correct classes when available', async () => {
```

Comment:
same here, why declaring the test as async?

Reply:
Same as https://github.com/Kwiket/jets-seatmap-react-lib-pub/pull/33#discussion_r1943224388 🙂